tests/dotnet/UnitTests/ProjectTest.cs

```diff
@@ -875,6 +875,10 @@ public void BuildProjectsWithExtensions (ApplePlatform platform)
 
 			var extensionPath = Path.Combine (Path.GetDirectoryName (consumingProjectDir)!, "bin", "Debug", platform.ToFramework (), GetDefaultRuntimeIdentifier (platform), "MySimpleApp.app", GetPlugInsRelativePath (platform), "ExtensionProject.appex");
 			Assert.That (Directory.Exists (extensionPath), $"App extension directory does not exist: {extensionPath}");
+
+			var pathToSearch = Path.Combine (Path.GetDirectoryName (consumingProjectDir)!, "bin", "Debug");
+			string [] configFiles = Directory.GetFiles (pathToSearch, "*.runtimeconfig.*", SearchOption.AllDirectories);
+			Assert.That (configFiles.Length > 0, "runtimeconfig.json file does not exist");
```

Comment:
Minor in this case, but it will give a slightly better assert message if the assert fails (it will say something like "expected not equal to zero" instead of "assertion failed")

```suggestion
			Assert.AreNotEqual (0, configFiles.Length, "runtimeconfig.json file does not exist");
```

alternatively this works too:

```suggestion
			Assert.That (configFiles.Length, Is.GreaterThan (0), "runtimeconfig.json file does not exist");
```